src/style.css

```diff
@@ -1,3 +1,77 @@
+/* stylelint-disable font-family-name-quotes */
+
+:root {
+  --main-blue: #00acdc;
+}
+
 body {
   margin: 0;
+  padding: 0;
+  font-family: 'Roboto', sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  background-color: gray;
+}
+
+header {
+  max-width: 100%;
+  background: #fff;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  position: relative;
+  padding: 0 50px;
+}
+
+.logo img {
+  height: 40px;
+}
+
+.navbar {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 60px;
+}
+
+.navbar-nav {
+  list-style: none;
+  display: flex;
+  height: 100%;
+}
+
+.nav-item {
```

Comment:
Be consistent with your vertical margins. Here you have a left margin for `.nav-item`. If you want to have spacing between items, it's better to choose either left or right margin consistently. Moreover, using `:not(:first-child)` pseudo-class could help you avoid resetting the margin on the first item.